annotations/views/rest_views.py

```diff
@@ -321,6 +333,93 @@ def create(self, request, *args, **kwargs):
+            if not interpretation_uri:
+                return Response(
+                    {"error": "An appellation must have a concept (interpretation) to be saved."},
+                    status=status.HTTP_400_BAD_REQUEST
+                )
+            
+            # Get the position data
+            position_value = request.data.get('position', {}).get('position_value')
+            if position_value:
+                # Update position
+                if not instance.position:
+                    instance.position = DocumentPosition.objects.create(
+                        occursIn_id=request.data['position']['occursIn'],
+                        position_type=request.data['position']['position_type'],
+                        position_value=position_value
+                    )
+                else:
+                    instance.position.position_value = position_value
+                    instance.position.save()
+
+            # Update string representation
+            if 'stringRep' in request.data:
+                instance.stringRep = request.data['stringRep']
+            
+            # Update interpretation
+            try:
+                concept = Concept.objects.get(uri=interpretation_uri)
+                instance.interpretation = concept
+            except Concept.DoesNotExist:
+                return Response(
+                    {"error": "The provided concept (interpretation) does not exist."},
+                    status=status.HTTP_400_BAD_REQUEST
+                )
+            
+            instance.save()
+            
+            serializer = self.get_serializer(instance)
+            return Response(serializer.data)
+            
+        except Exception as e:
+            logger.error(f"Error updating appellation: {str(e)}", exc_info=True)
+            return Response(
+                {"error": str(e)},
+                status=status.HTTP_400_BAD_REQUEST
+            )
+
+    def destroy(self, request, *args, **kwargs):
+        try:
+            instance = self.get_object()
+            logger.debug(f"Attempting to delete appellation {instance.id}")
+            
+            # Check if this Appellation is part of a Relation
+            in_relations = (
+                RelationSet.objects.filter(
+                    constituents__in=Relation.objects.filter(
+                        Q(source_content_type=ContentType.objects.get_for_model(Appellation), 
+                          source_object_id=instance.id) |
+                        Q(object_content_type=ContentType.objects.get_for_model(Appellation),
+                          object_object_id=instance.id) |
+                        Q(predicate=instance)
+                    )
+                ).exists()
+            )
+            if in_relations:
+                return Response(
+                    {"detail": "Cannot delete an Appellation used in a Relation."},
+                    status=status.HTTP_409_CONFLICT
```

Comment:
conflicts are typically send with there is a version of the sent resource that is different than the current one in the system (or different from what the sender based their changes on), so more like a GitHub conflict. Here I would send a bad request (since this should not be allowed in the first place to delete appellations used in a relation).